cmake/onnxruntime_unittests.cmake

```diff
@@ -1,5 +1,8 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+function(enable_post_merge_testing)
```

Comment:
I guess this function is not needed. If you removed it, the build should still work well.

Reply:
Let me try